System/Process.hsc

```diff
@@ -1,12 +1,10 @@
 {-# LANGUAGE CPP, ForeignFunctionInterface #-}
-#ifdef __GLASGOW_HASKELL__
```

Comment:
Shouldn't this still be here? Unless `InterruptibleFFI` is a portable extension now? or process only supports GHC?